ansible/roles/django/tasks/main.yml

```diff
@@ -0,0 +1,18 @@
+---
+- block:
+    - name: Install required packages
+      apt: pkg={{ item }} state=installed update-cache=yes
+      with_items:
+       - libmysqlclient-dev
+
+    - name: Install python requirements
```

Comment:
The intention was to make `django` a role that independently sets up a Django install that can be used by any playbook. The truth is that the role has a mix of frameworks which are not strictly needed for a simple Django app(tastypie, gunicorn, swagger). I'll just leave the Django dependency here for now.